### PR TITLE
[Lottie Exporter] Update documents for Outline layer

### PR DESCRIPTION
**Pull request by AnishGG**
This P.R. updates the outline layer and adds support for invert parameter in `region` layer, `outline` layer, and `polygon` layer.
@morevnaproject please have a look. ^__^

### CODE REVIEW

File: docs/projects/lottie/layers/polygon.rst [hunk 1]
```diff
@@ -1,32 +1,33 @@
...
+|  Vertices List  |             layers/solid.json -> helpers/mask.json -> "pt"             |
++-----------------+------------------------------------------------------------------------+
 
-.. note::
-    "it" is used for shapes within groups. When only a single shape is there, "shapes" will be used
-    "ty" = "fl" describes that the type of shape used is 'fill'.
+Important points
+----------------
+
+- Instead of directly using the shapes layer from Lottie, a solid color layer is used and is masked(using masks) furthur to draw a polygon. This is done to support the invert parameter from Synfig.
```
Comment (ice0) [inline docs/projects/lottie/layers/polygon.rst]: further?

File: docs/projects/lottie/layers/region.rst [hunk 2]
```diff
@@ -1,32 +1,33 @@
...
+|     Vertices    |             layers/solid.json -> helpers/mask.json -> "pt"             |
++-----------------+------------------------------------------------------------------------+
 
-.. note::
-    "it" is used for shapes within groups. When only a single shape is there, "shapes" will be used
-    "ty" = "fl" describes that the type of shape used is 'fill'.
+Important points
+----------------
+
+- Instead of directly using the shapes layer from Lottie, a solid region layer is used and is masked(using masks) furthur to draw a polygon. This is done to support the invert parameter from Synfig.
```
Comment (ice0) [inline docs/projects/lottie/layers/region.rst]: same here

**Comment by AnishGG** [comment]
@ice0 @morevnaproject 
Please review this P.R. I think it is ready to be merged.
I will add documents for remaining layers(rotate, scale, translate, group) in a seperate P.R. after the resolution of this one.
Thank you!

**Comment by morevnaproject** [comment]
Looks good! Thank you! ^__^

**Pull request merged**